types/webpack/index.d.ts

```diff
@@ -103,6 +104,8 @@ declare namespace webpack {
         [name: string]: string | string[];
     }
 
+    type EntryFunc = () => string | string[] | Promise<string | string[]>;
```

Comment:
Not sure if this is `(() => string) | string[]...` or `() => (string |  string[]...)`. Please add parenthesis to make it clear and possibly correct

Reply:
@mohsen1  Thanks for pointing it out.